docs/articles/faq/README.md

```diff
@@ -259,4 +262,27 @@ and [t.resizeWindow](../documentation/test-api/actions/resize-window.md#setting-
 to control the browser window size. If the test passes, it means your webpage hides
 the target element when the window is resized to smaller dimensions.
 
-Finally, try updating TestCafe to the latest version to see if the problem persists.
\ No newline at end of file
+Finally, try updating TestCafe to the latest version to see if the problem persists.
+
+### What are the possible reasons for an HTTP request failure? How can I address them?
+
+#### The Web server is not responding
+
+Check if the Web and DNS servers are online and configured to accept requests to this URL.
+
+#### Unstable or improperly configured network connection
+
+* Check the proxy server's, router's and network connection's settings.
+* Connect to a different network.
+* Restart or replace the router.
```

Comment:
So should I go and buy another router to check this hypothesis?

Reply:
Or you can connect your machine to the network directly.

Reply:
Probably we can say more common like "Check your network devices/settings".